integration_tests/cli/auth-demo.sh

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
```

Comment:
Do we have tests for negative/malicious cases:
1) The returned number of items (all accounts) isn't equal to the total number (of accounts)?
2) Invalid element is returned as part of the range request?
3) The returned item(s) are valid, but not within the range.
4) Not all elements from the range are returned.